src/include/splash/basetypes/ColTypeDim.hpp

```diff
@@ -88,7 +88,13 @@ namespace splash
             else
                 return NULL;
         }
-    };
+
+        static size_t getSize_()
```

Comment:
ugh - can you at least put this in private scope?
I don't fully see why this is necessary, the compile error just looked like a missing scope to me.